.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -51,11 +53,20 @@ jobs:
         with:
           path: OpenSearch-Dashboards/plugins/dashboards-observability
 
+      - name: Load Plugin Bootstrap cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+            **/target
+          key: ${{ runner.os }}-bootstrap-${{ hashFiles('OpenSearch-Dashboards/yarn.lock', 'OpenSearch-Dashboards/plugins/dashboards-observability/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-bootstrap-
 
       - name: Plugin Bootstrap
         run: |
-          cd OpenSearch-Dashboards/plugins/dashboards-observability
-          yarn osd bootstrap
+          cd OpenSearch-Dashboards
+          yarn config set network-timeout 1000000 -g
+          yarn osd bootstrap || yarn osd bootstrap
```

Comment:
this is ok but there's also a retry workflow https://github.com/marketplace/actions/retry-step